packages/dds/tree/src/core/rebase/utils.ts

```diff
@@ -74,6 +89,18 @@ export interface BranchRebaseResult<TChange> {
 	 * Details about how the commits on the source branch changed
 	 */
 	readonly commits: RebasedCommits<TChange>;
+	/**
+	 * Telemetry properties for the rebase operation.
+	 */
+	readonly telemetryProperties?: TelemetryProperties;
+}
+
+interface RebaseChangeResult<TChange> {
+	readonly change: TaggedChange<TChange>;
```

Comment:
@yann-achard-MS, in this [comment](https://github.com/microsoft/FluidFramework/pull/21282#discussion_r1626289545), you suggested creating an object containing a `TaggedChange`.

The `rebaseChangeOverChanges()` returns `TChange`, in this case, should I have `TChange` instead of `TaggeedChange<TChange>`?

Reply:
Yes, `TChange` is correct.